src/jit/flowgraph.cpp

```diff
@@ -21674,6 +21674,13 @@ GenTreePtr Compiler::fgAssignStructInlineeToVar(GenTreePtr child, CORINFO_CLASS_
         // If inlinee was just a call, new inlinee is v05 = call()
         newInlinee = gtNewAssignNode(dst, src);
 
+        // When returning the call returns in multiple registers, mark the variabale where
+        // the result is returned as MultiReg return, so it doesnot get promoted.
```

Comment:
You might want to simplify this comment as follows

_In var=call, where call returns its value in multiple registers, mark var so that it doesn't get struct promoted._